src/CMakeOptions.cmake

```
@@ -34,8 +34,11 @@ endif()
 add_compile_definitions(LF_DRKEY_FETCHER=${LF_DRKEY_FETCHER})
 add_compile_definitions(LF_DRKEY_FETCHER_${LF_DRKEY_FETCHER}=1)
 
-set(LF_CBCMAC "OPENSSL" CACHE STRING "CBC MAC implementation (OPENSSL).")
-if(LF_CBCMAC MATCHES "OPENSSL")
+set(LF_CBCMAC "OPENSSL" CACHE STRING "CBC MAC implementation (AESNI, OPENSSL).")
+if(LF_CBCMAC STREQUAL "AESNI")
```

Have default as first value in the description. I think that's pretty much the convention here.

```suggestion
set(LF_CBCMAC "OPENSSL" CACHE STRING "CBC MAC implementation (OPENSSL, AESNI).")
if(LF_CBCMAC STREQUAL "AESNI")
```

---

Done